src/XMakeCommandLine/Microsoft.Build.CommonTypes.xsd

```diff
@@ -213,6 +213,29 @@ elementFormDefault="qualified">
       </xs:complexContent>
     </xs:complexType>
   </xs:element>
+  <xs:element name="DotNetCliToolReference" substitutionGroup="msb:Item">
+    <xs:annotation>
+      <xs:documentation>
+        <!-- _locID_text="DotNetCliToolReference" _locComment="" -->A project-specific tool that can be ran from the command line. 
+      </xs:documentation>
+    </xs:annotation>
+    <xs:complexType>
+      <xs:complexContent>
+        <xs:extension base="msb:SimpleItemType">
+          <xs:sequence>
+            <xs:element name="Version" />
+          </xs:sequence>
+          <xs:attribute name="Include" type="xs:string">
```

Comment:
Short version - No it's not needed but I'll add it for a better experience.

Long version - Include/Exclude/Remove/Condition/Label show up from inheritance (from things in ItemGroups). You only need to provide it if you want to override the documentation for it. That being said the default message for any of them apply to these (i.e. "Semi-colon separated list of files...").

Looking at other entries in the file only Include is overridden which makes sense given that is the most common one but if we want to improve all usage of this we may want to consider overriding the rest as well.